bigchaindb/util.py

```diff
@@ -501,6 +501,26 @@ def get_hash_data(transaction):
     return crypto.hash_data(serialize(tx))
 
 
+def verify_vote_signature(signed_vote):
```

Comment:
Verifying the signature is not enough. We also need to check if a signature was created by a valid voter, so we need to check if the `node_pubkey` is in the `voters`.

Maybe have another method called `validate` block that checks if the `node_pubkey` is valid and then check the signature.